### DISCUSSION TITLE
HTML-webview element locate function

### DISCUSSION THREAD

**Discussion by zol9527** [ProxymanApp/Proxyman#2104] [Ideas]
I often need to check html files, but i feel it's not convenient to check where is the element.

**Comment by NghiaTranUIT** [comment]
You can command + F and find your element?

**Reply by zol9527** [reply]
Yeah. I can search it trough "cmd + f", but I feel more convenient to move mouser on an element, and press "cmd + shift + c" in browser, then I can directly locate the element.

If I use search function, there are more steps need to use, I need to select and copy characters then paste them to search, sometimes many results will appear to disturb my focus.Or I need to switch browser and proxyman frequently.

This function is very convenient for me, not everyone. I just hope this project can add this feature.
